elements/pfe-icon/src/pfe-icon.scss

```
@@ -130,11 +132,14 @@ $sizing: (
 // May need Feedback Darkest colors too.
 
 :host([circled]:not([circled="false"])) {
-    --pfe-icon--BackgroundColor: #{pfe-local(color, $fallback: pfe-var(surface--lightest))};
-    --pfe-icon--Color:           #{pfe-broadcasted(text)};
-    --pfe-icon--Padding:         .5em;
-    --pfe-icon--BorderWidth:     #{pfe-var(ui--border-width)};
-    --pfe-icon--BorderColor:     #{pfe-local(color, $fallback: pfe-var(surface--border))};
+  padding: pfe-local(Padding, $fallback: .5em);
+  border-width: pfe-local(BorderWidth, $fallback: pfe-var(ui--border-width));
+  border-color: pfe-local(color, $fallback: pfe-var(surface--border));
+  background-color: pfe-local(BackgroundColor, pfe-local(color, $fallback: pfe-var(surface--lightest)));
+
+  filter feFlood {
+    flood-color: pfe-local(Color, $fallback: pfe-broadcasted(text));
```

```suggestion
    flood-color: pfe-local(color, $fallback: pfe-broadcasted(text));
```

This component is using a lowercase `color` instead because it's the concept of color rather than the literal `color` property in CSS.

---

Finding that very confusing, it doesn't seem to be documented in the code or the README, is that something someone who understands it can write down the differences between the vars?

---

Yeah, now that Kyle is close to merging the documentation update, Kendall & I are working on bringing over the write-up but the gist is BEM.  In BEM, a property that is specific to a CSS property uses this format: BackgroundColor vs. background-color or backgroundColor.  It's how you can separate the idea of a color variable from a variable that literally is assigned to the color property in CSS.  Since this variable is assigned to flood-color, the options were: FloodColor or a more abstract naming.  Since you don't really need to know how the inner workings of the SVG function to style this component, `--pfe-icon--color` was chosen.

It will all be written out but for now, the summary is: `--pfe-icon--color` is correct over `--pfe-icon--Color`.

---

It's also in the README: 

```
For more fine-grained control, `--pfe-icon--color` is available to override the color of a specific icon or sets of icons and will be applied to either the SVG lines or the background of the circle (if circled).
```

---

Oh, damn. I thought it was, but then I checked and skimmed over it. 😵 Resolving convo